Lusas_oM/Lusas_oM.csproj

```diff
@@ -134,6 +134,10 @@
       <Private>False</Private>
       <SpecificVersion>False</SpecificVersion>
     </Reference>
+    <Reference Include="Geometry_oM, Version=6.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\..\..\..\..\ProgramData\BHoM\Assemblies\Geometry_oM.dll</HintPath>
```

Comment:
This will fail compliance checks - see other paths in the file.

Reply:
Changed to follow other References.